docs/_includes/options/events/jquery.html

```diff
@@ -43,8 +43,17 @@ <h2 id="events-public">
   <h3>
     Does Select2 include extra information in these events?
   </h3>
+If your server part returns additional data in the json dictionary, you can access like this:
 
-  {% include options/not-written.html %}
+{% highlight js linenos %}
+$(function() {
+    $('#mywidget').on('select2:select', function(evt){
+        alert(evt.params.data.mydata);
```

Comment:
I am happy that I found `evt.params.data`. I would like to share this information and provided this pull request. Please alter my pull request, if you think it can be improved. How do you want this change to look like?